README.md

```diff
@@ -30,3 +31,30 @@ function mystery(n) {
 Add your answer to this markdown file. [This
 page](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/writing-mathematical-expressions)
 might help with the notation for mathematical expressions.
+
+
+Recurrence relation for mystery():
+
+$T(n) =$ 1 if $n <= 1$ <br>
+$3T(\frac{n}{3}) + n^5$ otherwise <br>
+
+Solving by substitution:<br>
+$T(n) = 3T(\frac{n}{3}) + n^5$<br>
+$T(n) = 3(3T(\frac{n/3}{3}) + \frac{n}{3}^5) + n^5$<br>
+$T(n) = 9T(\frac{n}{9}) + 2n^5$<br>
```

Comment:
$3\left(\left(\frac{n}{3}\right)^5\right) \neq n^5$

Reply:
Ah I see. Corrected the math (hopefully). Let me know if this is on the right track!!